### ISSUE TITLE
acc

### ISSUE THREAD

**Issue by bingdidi** [GuHongyang/VaDE-pytorch#9]
After pre training, what is the prediction accuracy? My acc is only 0.7%. why?

**Comment by Ubiquinone-dot** [comment]
Pre-training does not organise the latent space very well so is unlikely to give a good accuracy to begin with. If you changed starting parameters / setting you might also run into vanishing gradients which I did, where the encoder doesn't change weights / encodings wrt to the initial random embedding, giving a low accuracy. You can check these things qualitatively by logging a t-SNE plot of the embedding space at each epoch during pretraining
